### DISCUSSION TITLE
Personal Dashboard falsely shows "Start Writing Code" section

### DISCUSSION THREAD

**Discussion by timokoenig** [community/community#15404] [Other Feature Feedback, Questions, & Ideas]
I just noticed a new personal dashboard and it falsely shows a "Start Writing Code" section with the reason `You're seeing this because you haven't created a repository in a while.` which is clearly not true.

It's not a big deal to dismiss these sections, but IMO this should not appear for experienced users in the first place.

![Screen Shot 2022-04-22 at 17 07 19](https://user-images.githubusercontent.com/4365095/164744260-83b50d90-fdfd-4c31-9c44-a4141134a060.png)

**Comment by ysfchn** [comment]
I just saw this page and shocked for a moment because I thought GitHub or someone deleted my all data and that's why I'm seeing a welcome page. 😅 I think this page should be displayed for new users, not for existing ones.

**Comment by nh-99** [comment]
I'm in complete agreement. I've been using Github since 2013 (almost 10 years) with public open-source repository contributions, and these sections showed up for me because I "haven't created a new repo" or "new pull request" in "a while".

I think it makes total sense for new users, but I am not a new user.

**Comment by MattIPv4** [comment]
The third section showing up openly implies that I've not really used GitHub before, but I _definitely_ have. Feels like these sections are just showing up without actually checking what they say they're checking?

![image](https://user-images.githubusercontent.com/12371363/164801523-97ae995f-ebbd-4098-a6b4-a967997b6a1d.png)

**Comment by LasseRafn** [comment]
Took you 10 years to tell me to get started.. I've been waiting for instructions 😜
